docs/src/auto_examples/core/sg_execution_times.rst

```diff
@@ -5,10 +5,10 @@
 
 Computation times
 =================
-**00:03.242** total execution time for **auto_examples_core** files:
+**02:36.161** total execution time for **auto_examples_core** files:
 
 +--------------------------------------------------------------------------------------------------------------+-----------+---------+
-| :ref:`sphx_glr_auto_examples_core_run_corpora_and_vector_spaces.py` (``run_corpora_and_vector_spaces.py``)   | 00:03.242 | 48.2 MB |
+| :ref:`sphx_glr_auto_examples_core_run_corpora_and_vector_spaces.py` (``run_corpora_and_vector_spaces.py``)   | 02:36.161 | 40.1 MB |
```

Comment:
This is weird. From 3 seconds to 2.5 minutes? What happened?

Reply:
I'm not sure. I ran the build again on my linux dev machine (the previous one was on MacOS) and the time is more appropriate (5s).